bin/tx-prover/prometheus.yml

```diff
@@ -0,0 +1,11 @@
+global:
+  scrape_interval: 15s # Set the scrape interval to every 15 seconds. Default is every 1 minute.
+  evaluation_interval: 15s # Evaluate rules every 15 seconds. The default is every 1 minute.
+  # scrape_timeout is set to the global default (10s).
+
+# A scrape configuration containing exactly one endpoint to scrape:
+scrape_configs:
+  - job_name: "proxy"
```

Comment:
Is "proxy" name somehow significant here? (e.g., could I rename it to something else and would everything still work fine?). Let's add a comment clarifying this.

Reply:
Ok! Will add a comment. 

It is not significant but just a label.